tigerpath/majors_and_certificates/majors/CBE.json

```diff
@@ -484,7 +542,7 @@
             ]
         },
         {
-            "name": "Technical Diversity",
```

Comment:
https://cbe.princeton.edu/sites/cbe/files/undergrad_handbook.pdf

Reply:
Everything below here is the same as above, right? So for any changes you make above, also make them below.